fastlane/lib/fastlane/actions/ensure_bundle_exec.rb

```diff
@@ -0,0 +1,61 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    # Raises an exception and stop the lane execution if the repo is not on a specific branch
+    class EnsureBundleExecAction < Action
+      def self.run(params)
+        gemfile_path = PluginManager.new.gemfile_path
+        if gemfile_path
```

Comment:
This could be `return if gemfile_path.nil?` instead of having an `if`

Reply:
✅